src/Pages/ProductDetail/Components/BottomBar.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import * as configs from "../../../config.js";
+
+class BottomBar extends Component {
+  constructor() {
+    super();
+    this.cart = React.createRef();
+    this.state = {
+      quantityValue: 1,
+      totalPrice: "",
+    };
+  }
+
+  componentDidMount() {
+    fetch("data/BottomBar.json")
+      .then((res) => res.json())
+      .then((res) => {
+        this.setState({
+          productPrice: res.ProductData[0].price,
+        });
+      });
+  }
+
+  handleDecrease = () => {
+    const totalSubtraction = this.state.totalPrice - this.state.productPrice;
+    this.setState({
+      quantityValue: this.state.quantityValue - 1,
+      totalPrice: totalSubtraction,
+    });
+  };
+
+  handleIncrease = () => {
+    const totalAdded = Number(this.state.productPrice) * this.state.quantityValue;
```

Comment:
```suggestion
    const totalAdded = +this.state.productPrice * this.state.quantityValue;
```
number로 바꿔줄 때 이런 표현도 가능합니다.!

Reply:
확인하였습니다